pyicloud/services/drive.py

```diff
@@ -304,20 +319,14 @@ def get(self, name):
     def __getitem__(self, key):
         try:
             return self.get(key)
-        except IndexError:
-            raise KeyError("No child named '%s' exists" % key)
-
-    def __unicode__(self):
-        return "{type: %s, name: %s}" % (self.type, self.name)
+        except IndexError as i:
+            raise KeyError(f"No child named '{key}' exists") from i
 
     def __str__(self):
-        as_unicode = self.__unicode__()
-        if PY2:
-            return as_unicode.encode("utf-8", "ignore")
-        return as_unicode
+        return fr"\{type: {self.type}, name: {self.name}\}"
```

Comment:
`fr` ?

Reply:
`f` as in f-string, and `r` as in raw string.